ui/imports/shared/popups/send/panels/RecipientSelectorPanel.qml

```diff
@@ -192,31 +190,15 @@ Item {
                             text: LocaleUtils.currencyAmountToLocaleString(entry.amountCurrency)
                         }
                     ]
-                    onClicked: recipientSelected(entry, TabAddressSelectorView.Type.RecentsAddress)
+                    onClicked: recipientSelected(entry, Helpers.RecipientAddressObjectType.RecentsAddress)
```

Comment:
Probably not part of this PR, but signals like this should pass only unique key of given element, not all metadata needed on the receiver, external side. Within the component we don't know what is needed externally. It may be e.g. role from the model which is not used in this component.

Reply:
I totally agree but for this superficial cleanup I didn't go deeper to that. I will create a task for a next step: https://github.com/status-im/status-desktop/issues/15492